### PR TITLE
mosquitto: Set mosquitto.conf log_type based on logging level

### PR DESCRIPTION
**Pull request by b3nj1**
Hi @pvizeli ,
I have resubmitted https://github.com/home-assistant/addons/pull/1703 after fixing my git mistakes and with your recommendation to use the system level logging.

I can't manage to change the configured logging level to test it. Do you know where/how I should change to something besides my current setting of 'info'? I've tried this and rebooted. 

```
# configuration.yaml
logger:
  default: warn
```

- Benjamin

### CODE REVIEW

**Comment by b3nj1** [comment]
OK - I was able to test it and get a reduction in logging output with warning vs info.

It took me a while to figure this out, so adding some notes for future me or someone else looking around.

```
config $ ha supervisor options -l warning
```

**Comment by b3nj1** [comment]
Hi @pvizeli ,

The auth_srv.sh was using unguarded echo logging on each connection. I updated my PR to change this to use bashio::log.<level>.

At present, the logging level defaults to "info" in bashio. Should the [bashio initial log level](https://github.com/hassio-addons/bashio/blob/3bf226743133dc6a6126be9f49d55bff62f31100/lib/bashio.sh#L32) be changed to pick up the supervisor level?  

Thanks for any help,
Benjamin

**Comment by b3nj1** [comment]
I've updated the PR to set the bashio logging level to match the supervisor level. I've been using this for a while without issue. Please let me know if there anything else I can do to make this PR ready for acceptance.

Thanks,
Benjamin

**Comment by Erriez** [comment]
@b3nj1 I was looking for an easy method to configure `log_type all` in mosquitto addon and found your PR. What is the easiest way to configure `log_type all` with your method? Thanks!

File: mosquitto/data/auth_srv.sh [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+#!/usr/bin/env bashio
 # shellcheck disable=SC2244,SC1117
+set +u
 set -e
 
 CONFIG_PATH=/data/options.json
 SYSTEM_USER=/data/system_user.json
 REQUEST=()
 REQUEST_BODY=""
+LOGGING=$(bashio::info 'hassio.info.logging' '.logging')
```
Comment (pvizeli) [inline mosquitto/data/auth_srv.sh:10]:
```suggestion
LOGGING=$(bashio::info.logging)
```

**Review by pvizeli — COMMENTED** [review]
Please update the changelog + bump the version and rebase it current master to run the CI on it

**Review by pvizeli — APPROVED** [review]

**Pull request merged**